curriculum/challenges/english/01-responsive-web-design/css-balance-sheet/step-001.md

```diff
@@ -0,0 +1,126 @@
+---
+id: 6193d8081ec2531581ea7b98
+title: Step 1
+challengeType: 0
+dashedName: step-1
+---
+
+# --description--
+
+A balance sheet is used to track the state of an organization's finances. You'll be building a mock balance sheet for freeCodeCamp to explore the use of various CSS selectors.
+
+Begin with your basic HTML boilerplate. Include the `DOCTYPE` declaration, the appropriate `meta` tags, a `head`, `body`, and `title` element, and `link` your stylesheet.
+
+# --hints--
+
+Your code should have a `<!DOCTYPE html>` declaration.
+
+```js
+assert(code.match(/<!DOCTYPE html>/i));
+```
+
+Your code should have an `html` element.
+
+```js
+assert(document.querySelectorAll('html').length === 1);
```

Comment:
Nich....

Before I review this, do you have something you would like to add...?

Reply:
?.?.?.?.?.